go.mod

```diff
@@ -227,3 +227,5 @@ require (
 )
 
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.1-0.20240131200429-02120488a4c0
+
+replace github.com/containers/storage => github.com/Honny1/storage v0.0.0
```

Comment:
FYI you do not have to create tags on your fork.
The way to do this is to replace the tag with the branch name or commit sha and then just run make vendor. Go will figure it our from there and replace it with pseudo version, i.e.
`github.com/containers/common v0.59.1-0.20240603155017-49ad520556e7`

Reply:
Thank